### PR TITLE
docs(readme): add SpeedVitals to the list of integrations

### PR DESCRIPTION
**Pull request by kraten**
Add [SpeedVitals](https://speedvitals.com/) to the list of integrations

> SpeedVitals is a Lighthouse powered tool to measure Web Vitals across multiple devices and locations. It has various features like Layout Shift Visualization, Waterfall Chart, Field Data and Resource Graphs.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -351,6 +351,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[DTEKT.IO](https://dtekt.io)** - DTEKT is a website performance and uptime monitoring service. It uses lighthouse to provide visibility into the performance of websites from multiple locations on multiple devices. It offers three months free trial and paid plans.
 
+* **[SpeedVitals](https://speedvitals.com)** - SpeedVitals is a Lighthouse powered tool to measure Web Vitals across multiple devices and locations. It has various features like Layout Shift Visualization, Waterfall Chart, Field Data and Resource Graphs.
```
Comment (patrickhulce) [inline readme.md:354]: I think several folks would find the use of "Web Vitals" here problematic as the Core Web Vitals are only measured on a page's complete lifecycle on real-users which cannot be accomplished in the lab. 

Mind changing to something like?
```suggestion
* **[SpeedVitals](https://speedvitals.com)** - SpeedVitals is a Lighthouse powered tool to measure vital performance metrics across multiple devices and locations. It has various features like Layout Shift Visualization, Waterfall Charts, Field Data and Resource Graphs.
```

Comment (patrickhulce) [reply readme.md:354]: also if there are any intentions of turning this into a paid product, please mention that here :)

**Review by patrickhulce — CHANGES_REQUESTED** [review]

**Comment by patrickhulce** [comment]
thanks very much @kraten! neat service :)

**Review by patrickhulce — APPROVED** [review]
thanks!

**Comment by kraten** [comment]
@patrickhulce Thanks for reviewing! Glad you liked it :)

**Pull request merged**